### PR TITLE
feat: adds ssh url in action's check

### PR DESCRIPTION
**Pull request by prajjwaldimri**
- Also fixes tests

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,47 +1,47 @@
-name: 'ActionDebugger By Warpbuild'
-description: 'Debug your GitHub Actions Environment interactively by using SSH or a Web shell'
+name: "ActionDebugger By Warpbuild"
+description: "Debug your GitHub Actions Environment interactively by using SSH or a Web shell"
 branding:
   icon: terminal
   color: gray-dark
-author: 'Warpbuild'
+author: "Warpbuild"
```
Comment (abhijit-hota) [inline action.yml:6]: Let's replace all instances of `Warpbuild` with `WarpBuild`

File: action.yml [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
   sudo:
-    description: 'If apt should be executed with sudo or without'
+    description: "If apt should be executed with sudo or without"
     required: false
-    default: 'auto'
+    default: "auto"
   install-dependencies:
-    description: 'Whether or not to install dependencies on linux (openssh-client, xz-utils)'
+    description: "Whether or not to install dependencies on linux (openssh-client, xz-utils)"
     required: false
```
Comment (abhijit-hota) [inline action.yml:19]: What is the use case for this option?

File: src/helpers.js [hunk 3]
```diff
@@ -66,18 +70,122 @@ export const getValidatedInput = (key, re) => {
...
 /**
  * @return {Promise<string>}
  */
 export const getLinuxDistro = async () => {
   try {
-    const osRelease = await fs.promises.readFile("/etc/os-release")
-    const match = osRelease.toString().match(/^ID=(.*)$/m)
-    return match ? match[1] : "(unknown)"
+    const osRelease = await fs.promises.readFile("/etc/os-release");
+    const match = osRelease.toString().match(/^ID=(.*)$/m);
```
Comment (abhijit-hota) [inline src/helpers.js:81]: Prefer `/^ID=(.+)$/m` as it won't allow empty strings.

File: src/index.js [hunk 4]
```diff
@@ -236,6 +242,32 @@ export async function run() {
...
+    };
+    const authToken = core.getInput("github-token");
+    let runCheckID = "";
+    if (authToken) {
+      const octokit = github.getOctokit(authToken);
+      const sha = getSHA();
+      core.debug(
+        `Creating a new Run on ${ownership.owner}/${ownership.repo}@${sha}`
+      );
+      const checkName = `${tmateSSH}`;
```
Comment (abhijit-hota) [inline src/index.js:259]:
```suggestion
      const checkName = tmateSSH;
```

**Review by abhijit-hota — CHANGES_REQUESTED** [review]
Added some minor change requests from what I can see. A few more things that can be addressed:

- Can move the `os/distro` -> `package manager command` logic to a `composeInstallationCommand` function.
- Should we add support for the following? I think at least `redhat` should be added. 
    |   Distro   | Package manager |
    | :--------: | :-------------: |
    |  `redhat`  |      `yum`      |
    | `openSUSE` |     `zypp`      |
    |  `gentoo`  |    `emerge`     |
- Can move the `sleep` and `continueFileExists` functions to `helpers.js`.

File: action.yml (hunk 2 above)
Comment (prajjwaldimri) [reply action.yml:19]: If we are creating custom images, we can include these dependecies in the image itself, reducing the time to start tmate.

Comment (abhijit-hota) [reply action.yml:19]: Ah, cool.

**Review by abhijit-hota — APPROVED** [review]

**Pull request merged**